server/container/guice/cassandra-guice/src/test/java/org/apache/james/ESReporterTest.java

```diff
@@ -47,6 +47,7 @@
 import org.elasticsearch.search.builder.SearchSourceBuilder;
 import org.junit.jupiter.api.AfterEach;
 import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Disabled;
```

Comment:
Let's deprecate ES7 metric reporting and only rely on the prometheus endpoints, rather that some custom, unmaintained reporters?

Reply:
It seems reasonable to me to rely solely on prometheus

Reply:
Shall i remove it from the project?

Reply:
Maybe we can raise the topic on server-dev mailing list, explain that it do not work with ES v6, that the underlying library is unmaintained (gasp) and that we have a pronmetheus alternative.

Reply:
I'll give my :+1: there.

Reply:
I tried to add the question to the mailing list, but it did not work.

Reply:
https://www.mail-archive.com/server-dev@james.apache.org/msg69816.html